### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by Nesffer** [rgbkrk/atom-script#433]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (C:\Users\Jimin\AppData\Local\atom\app-0.198.0\resources\app.asar\src\atom.js:57:13)
Object.deactivate (C:\Users\Jimin\.atom\packages\script\lib\script.coffee:39:9)
Package.deactivate (C:\Users\Jimin\AppData\Local\atom\app-0.198.0\resources\app.asar\src\package.js:635:21)
PackageManager.deactivatePackage (C:\Users\Jimin\AppData\Local\atom\app-0.198.0\resources\app.asar\src\package-manager.js:468:12)
```

**Comment by erran** [comment]
@Nesffer Should be fixed in the latest release (2.21.0). cc @rgbkrk

**Comment by rgbkrk** [comment]
Please update script if you can, this is all fixed up now.

```
$ apm update script
Package Updates Available (1)
└── script 2.21.0 -> 2.22.0

Would you like to install these updates? (yes) yes

Installing script@2.22.0 to /Users/rgbkrk/.atom/packages ✓
```
